app/views/spree/admin/products/related.html.erb

```diff
@@ -12,33 +12,33 @@
     <fieldset>
       <legend><%= Spree.t(:add_related_product) %></legend>
       <div class="row">
-        <div id="related_product_name" class="">
-          <div class="form-group">
-            <%= label_tag :add_variant_name, Spree.t(:name_or_sku_short) %>
-            <%= hidden_field_tag :add_variant_name, '', class: 'variant_autocomplete', style: "display: block !important;" %>
-          </div>
-        </div>
+        <div class="alpha four columns">
```

Comment:
We're moving away from classnames like this towards using some of the Bootstrap stuff that was implemented a while ago. Unfortunately, the Bootstrap classes aren't available to us yet. Maybe we could add a stylesheet with some styles like this: https://github.com/solidusio/solidus/blob/master/backend/app/assets/stylesheets/spree/backend/sections/_products.scss#L1-L16